README.md

```diff
@@ -7,29 +37,68 @@ sc2.init({
   scope: ['vote', 'comment']
 });
 ```
+Parameters:
+- __app__: This is the name of the app that was registered in the SteemConnect V2 dashboard
+- __callbackURL__: This is the URL that users will be redirected to after interacting with SC2. It must be listed in the "Redirect URI(s)" list in the app settings EXACTLY the same as it is specified here
+- __accessToken__: If you have an oauth2 access token for this user already you can specify it here, otherwise you can leave it and set it later using sc2.setAccessToken(accessToken).
+- __scope__: This is a list of operations the app will be able to access on the user's account. For a complete list of scopes see: [https://github.com/steemit/steemconnect/wiki/OAuth-2#scopes](https://github.com/steemit/steemconnect/wiki/OAuth-2#scopes)
+
+### Get Login URL
+The following method returns a URL that you can redirect the user to so that they may log in to your app through SC2:
+```
+var link = sc2.getLoginUrl(state);
+console.log(link)
+// => https://v2.steemconnect.com/oauth2/authorize?client_id=[app]&redirect_uri=[callbackURL]&scope=vote,comment&state=[state]
+```
+Parameters:
+- __state__: Data that will be passed to the callbackURL for your app after the user has logged in.
+
+After logging in, SC2 will redirect the user to the "redirect_uri" specified in the login url above and add the following query string parameters for your app to use:
+- __accessToken__: This is the oauth2 access token that is required to make any Steem API calls on behalf of the current user. Once you have this you need to tell the SC2 SDK to use it by either specifying it as a parameter to the init() method call or by calling sc2.setAccessToken([accessToken]).
```

Comment:
@bonustrack it looks like "accessToken" is the name currently being used for this parameter everywhere in https://github.com/steemit/sc2-sdk/blob/dev/src/sc2.js For example:
```
sc2.setAccessToken = (accessToken) => { sc2.accessToken = accessToken; };
```
If "access_token" is used as a parameter to the init() method it will not work. Let me know if i'm missing something here.